command/src/main/java/com/jagrosh/jdautilities/command/Command.java

```diff
@@ -78,7 +78,7 @@
     protected String help = "no help available";
     
     /**
-     * The {@link com.jagrosh.jdautilities.command.Command.Category Category} of the command.
+     * The {@link Command.Category Category} of the command.
```

Comment:
See earlier comments on JDA documentation standards

Reply:
Same with this...